### PR TITLE
fix/PPTP-1814 | Content Update - What is your organisation type page

### PR DESCRIPTION
**Pull request by ajchilts**
### Description of Work carried through

Brief description of what/why/how.

#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [x] `sbt scalafmt test:scalafmt` was executed
 - [ ] Required Environment Config has been amended/added
 - [x] Links to dependencies have been included (BE/FE work)
 - [x] User Acceptance Tests (UAT) were run locally and have passed.
 - [x] No Accessibility errors/warnings reported by Wave

UAT pack PR: https://github.com/hmrc/plastic-packaging-tax-acceptance/pull/114

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -383,6 +383,7 @@ organisationDetails.type.ScottishLimitedPartnership = Scottish limited partnersh
 
 organisationDetails.type.GroupNominatedPartnership = Limited liability partnership
 organisationDetails.type.hint = Register for Plastic Packaging Tax if you are a charitable incorporated organisation (CIO), a trust, an overseas organisation without a UK establishment, or a public body.
+organisationDetails.type.href = https://www.tax.service.gov.uk/submissions/new-form/register-for-plastic-packaging-tax
```
Comment (PSouthern) [inline conf/messages.en:386]: url currently returns a 404 - is that expected?

File: app/uk/gov/hmrc/plasticpackagingtax/registration/views/group/organisation_type.scala.html [hunk 2]
```diff
@@ -57,6 +59,8 @@
     )
 }
 
+@linkHint = {<a href=@messages("organisationDetails.other.type.href") class="govuk-link" id="organisationDetails-other-type-link">@messages("organisationDetails.other.type.hint")</a>}
```
Comment (RyanKellyHMRC) [inline app/uk/gov/hmrc/plasticpackagingtax/registration/views/group/organisation_type.scala.html:62]: Not a showstopper by any means but you've brought the link component in and then handcrafted one.

File: conf/messages.en (hunk 1 above)
Comment (ajchilts) [reply conf/messages.en:386]: I don't know @PSouthern 🤔 . It was the url I was passed from Rebecca.

File: app/uk/gov/hmrc/plasticpackagingtax/registration/views/organisation/organisation_type.scala.html [hunk 3]
```diff
@@ -68,6 +70,8 @@
           )
     }
 
+    @linkHint = {<a href=@messages("organisationDetails.type.href") class="govuk-link" id="organisationDetails-type-link">@messages("organisationDetails.type.hint")</a>}
```
Comment (RyanKellyHMRC) [inline app/uk/gov/hmrc/plasticpackagingtax/registration/views/organisation/organisation_type.scala.html:73]: Same here.

File: conf/messages.en (hunk 1 above)
Comment (paulblakehmrc) [reply conf/messages.en:386]: that url will only go live on 31 March

**Review by RyanKellyHMRC — APPROVED** [review]
All good.

**Pull request merged**